academy/3-my-own-chain/migration.md

```diff
@@ -562,126 +1027,1210 @@ func getOrNewPlayerInfo(infoSoFar *map[string]*types.PlayerInfo, playerIndex str
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
+
+```sh
+$ docker exec -it checkers bash -c "cat <<< \$(jq '.app_state.gov.voting_params.voting_period = \"600s\"' /root/.checkers/config/genesis.json) > /root/.checkers/config/genesis.json"
+```
+
+</CodeGroupItem>
+
+</CodeGroup>
+
+You can confirm that the value is in using the earlier command.
+
+Make Alice the chain's validator too by creating a genesis transaction modeled on that done by Ignite, as found in `config.yml`:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local" active>
+
+```sh
+$ ./release/v1/checkersd gentx alice 100000000stake \
+    --keyring-backend test --chain-id checkers
+$ ./release/v1/checkersd collect-gentxs
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
+
+```sh
+$ docker exec -it checkers ./release/v1/checkersd gentx alice 100000000stake \
+    --keyring-backend test --chain-id checkers
+$ docker exec -it checkers ./release/v1/checkersd collect-gentxs
+```
+
+</CodeGroupItem>
+
+</CodeGroup>
+
+Now you can start the chain proper:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local" active>
+
+```sh
+$ ./release/v1/checkersd start
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
+
+```sh
+$ docker exec -it checkers ./release/v1/checkersd start \
+    --rpc.laddr "tcp://0.0.0.0:26657"
+```
+
+Note that you need to force the node to listen on all IP addresses, not just `127.0.0.1` as it would do by default.
+
+</CodeGroupItem>
+
+</CodeGroup>
+
+From another shell, create a few un-played games with:
+
+<CodeGroup>
+
+<CodeGroupItem title="Local" active>
+
+```sh
+$ export alice=$(./release/v1/checkersd keys show alice -a)
+$ export bob=$(./release/v1/checkersd keys show bob -a)
+$ ./release/v1/checkersd tx checkers create-game \
+    $alice $bob 10 stake \
+    --from $alice --keyring-backend test --yes \
+    --broadcast-mode block
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Docker">
+
+```sh
+$ export alice=$(docker exec checkers ./release/v1/checkersd keys show alice -a --keyring-backend test)
+$ export bob=$(docker exec checkers ./release/v1/checkersd keys show bob -a --keyring-backend test)
+$ docker exec -it checkers ./release/v1/checkersd tx checkers create-game \
+    $alice $bob 10 stake \
+    --from $alice --keyring-backend test --yes \
+    --broadcast-mode block
+```
+
+</CodeGroupItem>
+
+</CodeGroup>
+
+<HighlightBox type="note">
+
+The `--broadcast-mode block` flag means that you can fire up many such games by just copying the command without facing any sequence errors.
+
+</HighlightBox>
+
+To get a few complete games, you are going to run the [integration tests](https://github.com/cosmos/academy-checkers-ui/blob/server-indexing/test/integration/stored-game-action.ts) against it. These tests were built to run against a running chain started by Ignite. What is different here is that:
+
+1. Blocks come slower, likely every five seconds instead of every one.
```

Comment:
@xavierlepretre Should this be a numbered list or a bullet point one?

Reply:
I put numbers because the next list should say _respectively_.